core/src/main/scala/org/apache/spark/sql/crossdata/daos/EphemeralTableStatusDAO.scala

```diff
@@ -23,10 +23,10 @@ import org.apache.spark.sql.crossdata.models.EphemeralStatusModel
 import org.apache.spark.sql.crossdata.serializers.CrossdataSerializer
 
 trait EphemeralTableStatusDAO extends GenericDAOComponent[EphemeralStatusModel]
-with TypesafeConfigComponent with SparkLoggerComponent with CrossdataSerializer {
+with TypesafeConfigComponent with SparkLoggerComponent with CrossdataSerializer with PrefixedDAO {
 
   override implicit val formats = json4sJacksonFormats
 
-  override val dao: DAO = new GenericDAO(Option(EphemeralTableStatusPath))
+  override val dao: DAO = new GenericDAO(Option(prefix+EphemeralTableStatusPath))
```

Comment:
Same as above, it is not a matter of saving space or the like. Just that if your future yourself - o some other guy (or girl) - need to change the way the name is built then you, he or she will suffer while hammering the change all over the code.